rest/src/main/java/org/jboss/pnc/rest/endpoint/RunningBuildRecordEndpoint.java

```diff
@@ -66,10 +77,22 @@ public BuildRecordRest getSpecific(@ApiParam(value = "BuildRecord id", required
         return buildRecordProvider.getSpecificRunning(id);
     }
 
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successful retrieval of the log"),
+            @ApiResponse(code = 204, message = "RunningBuild exists, but the log content is empty"),
+            @ApiResponse(code = 404, message = "RunningBuild with specified id does not exist"),
+    })
     @ApiOperation(value = "Gets specific log of a Running Build Record")
     @GET
     @Path("/{id}/log")
-    public Response getLogs(@ApiParam(value = "BuildRecord id", required = true) @PathParam("id") Integer id) {
-        return Response.ok(buildRecordProvider.getLogsForRunningBuildId(id)).build();
+    public Response getLogs(@ApiParam(value = "RunningBuild id", required = true) @PathParam("id") Integer id) {
```

Comment:
ooh yes I see - we also have 404 for not found. Please ignore my previous comment.